docs/web/authentication.md

```diff
@@ -631,17 +631,25 @@ Personal tokens can be written instead of the user's password in the
   </summary>
 
 ```
-usage: CodeChecker cmd token new [-h] [--description DESCRIPTION]
+usage: CodeChecker cmd token new [-h] --name NAME [--description DESCRIPTION]
                                  [--url SERVER_URL]
                                  [--verbose {info,debug,debug_analyzer}]
 
 Creating a new personal access token.
 
-optional arguments:
+options:
   -h, --help            show this help message and exit
+  --name NAME           A unique name that identifies the access token.
+                        (default: None)
```

Comment:
Wouldn't it make sense to make this a positional argument, same as when deleting a token?

Reply:
Done